packages/components/cypress/e2e/menu-button.cy.ts

```diff
@@ -56,6 +56,7 @@ describe('menus', { baseUrl: null, includeShadowDom: true }, () => {
       cy.get('post-menu-trigger[for="menu-three"]').eq(1).as('triggerD');
       cy.get('post-menu#menu-three').as('menuC');
       cy.get('post-menu#menu-four').as('menuD');
+      cy.wait(500);
```

Comment:
That does not seem like the proper way to handle this. An element selector would be more appropriate since cypress is waiting for it, for example `cy.get('post-menu#menu-four[data-hydrated]')` or something along those lines.